interface/main/calendar/modules/PostCalendar/pntemplates/default/user/ajax_search.html

```diff
@@ -4,8 +4,8 @@
 [-* we want to include our stylesheet for this view*-]
 <link rel="stylesheet" href="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-datetimepicker/build/jquery.datetimepicker.min.css">
 
-<!--<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery/dist/jquery.min.js"></script>-->
-<script type="text/javascript" src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-datetimepicker/build/jquery.datetimepicker.full.min.js"></script>
+<!--<script src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery/dist/jquery.min.js"></script>-->
+<script src="[-php-] echo $GLOBALS['assets_static_relative'] [-/php-]/jquery-datetimepicker/build/jquery.datetimepicker.full.min.js"></script>
 
```

Comment:
Hello @bradymiller and @tywrenn 

Can I remove this comment script?

Reply:
@stu01509 Go ahead since it's commented out

Reply:
@tywrenn Updated the commit.